src/social/interface/pagination.dto.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  IsUUID,
+  IsOptional,
+  IsNumberString,
+  IsInt,
+  IsString,
+} from 'class-validator';
+import { Transform } from 'class-transformer';
+
+export class DtoPagination {
+  @IsOptional()
+  @IsString()
+  after?: string;
+
+  @IsOptional()
+  @IsString()
+  before?: string;
+
+  @IsOptional()
+  @Transform(parseInt)
+  @IsInt()
+  limit?: number;
+}
```

Comment:
Only specify 1 of:
- after (next page)
- before (prev page)

after/before takes commentId
if no after/before is specify -> return all 
limit (optional): how many comments to retrieve